permissions/plugin-screenrecorder.yml

```
@@ -0,0 +1,7 @@
+---
+name: "screenrecorder"                                                                                                                                                                                             
+github: "jenkinsci/screenrecorder-plugin"
+paths:
+- "org/jenkins-ci/plugins/screenrecorder"
```

```suggestion
- "io/jenkins/plugins/screenrecorder"
```

Should be based on the `<groupId></groupId>` tag from your pom.xml. See https://github.com/jenkinsci/screenrecorder-plugin/blob/master/pom.xml#L12